incubator/jupyter-notebook/jupyter-notebook/templates/deployment.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "jupyter.fullname" . }}
+  labels:
+    app: {{ template "jupyter.name" . }}
```

```suggestion
    app: {{ template "jupyter.name" . }}
    chart: {{ template "jupyter.chart" . }}
    release: {{ .Release.Name }}
    instance: {{ .Values.Instance | quote }}
```

Add SLATE labeling scheme